configs/nfv.yaml

```
@@ -28,15 +28,15 @@ sriov_nova_pci_passthrough:
     physical_network: "mellanox-sriov"
 sriov_nic_numvfs: 8
 dpdk_interface: enp193s0f0
-kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=420 iommu=pt amd_iommu=on isolcpus=10-23,58-71,34-47,82-95"
-tuned_isolated_cores: 10-23,58-71,34-47,82-95
+kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=420 iommu=pt amd_iommu=on isolcpus=15-23,63-71,39-47,87-95"
+tuned_isolated_cores: 15-23,63-71,39-47,87-95
 extra_heat_params:
   CinderRbdFlattenVolumeFromSnapshot: true
-  NovaComputeCpuDedicatedSet: "10-23,58-71,34-47,82-95"
+  NovaComputeCpuDedicatedSet: "16-23,63-71,40-47,87-95"
   NovaReservedHostMemory: 4096
-  NovaComputeCpuSharedSet: "0-9,48-57,24-33,72-81"
+  NovaComputeCpuSharedSet: "0-14,48-62,24-38,72-86"
```

This is a nit, but this gives us 34 pCPUs. We only need 32 (assuming our flavor uses 8 vCPUs). How about:

```suggestion
  NovaComputeCpuDedicatedSet: "16-23,64-71,40-47,88-95"
  NovaReservedHostMemory: 4096
  NovaComputeCpuSharedSet: "0-14,48-63,24-38,72-87"
```
```

---

Ok!